### ISSUE TITLE
WebM last frame missing

### ISSUE THREAD

**Issue by crossperia** [fnordware/AdobeWebM#25]
Hello,

I using this plugin with Premiere/AfterEffects CS6,
and found some WebM lost the last frame.

Example:
convert 640 TIFF images to WebM by ffmpeg (set fps 30)

In the web browser(Firefox), this is nice.
But in Premiere/AfterEffects, it seems to miss the last frame.

I try some fps, and the results is below.
frames| fps | duration
 640  | 15  | 00:42:10
 639  | 30  | 00:21:09
 640  | 30  | 00:21:09 (!?)
 641  | 30  | 00:21:10 (!?)
 640  | 60  | 00:10:40

Is this caused by plugin or ffmpeg?

**Comment by fnordware** [comment]
Hmmm, could you post your WebM somewhere for me to see?

**Comment by crossperia** [comment]
Hello,

I could not attach the WebM file, because of file type error.
So, I changed the file extension webm to pdf...
Can you download this correctly?
[sample.webm.pdf](https://github.com/fnordware/AdobeWebM/files/36297/sample.webm.pdf)
Please rename this to "sample.webm"

sample.webm [frames:640(0~639), fps:30]

**Comment by fnordware** [comment]
OK, thanks for the bug report. I've fixed your missing frame problem by doing a little rounding on the video duration (52380d742a4ee78674277a6ef4dc1ca2439909c3). The fix will be in a future build.

In the course of looking at this, I found that the plug-in is not writing the duration correctly due to a bug in another library. I filed a bug with Google [here](https://bugs.chromium.org/p/webm/issues/detail?id=1100).
